From 3d051e7c79897b91ec1e35f7b6a341e36910ab4d Mon Sep 17 00:00:00 2001
From: Nitin Garg <nitin.garg@nxp.com>
Date: Tue, 13 Feb 2018 21:25:40 -0600
Subject: [PATCH 3362/5242] MLK-17597: Add GICC, GICH, GICV addresses to
 iMX8QM device tree.

commit  8db59b65bbc4672d337e38194a6b9b36ce1c38b8 from
https://source.codeaurora.org/external/imx/linux-imx.git

Adding GICC, GICH, GICV addresses for iMX8QM

Signed-off-by: Nitin Garg <nitin.garg@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 9dc8091..903bf40 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -92,7 +92,10 @@
 	gic: interrupt-controller@51a00000 {
 		compatible = "arm,gic-v3";
 		reg = <0x0 0x51a00000 0 0x10000>, /* GIC Dist */
-		      <0x0 0x51b00000 0 0xC0000>; /* GICR (RD_base + SGI_base) */
+		      <0x0 0x51b00000 0 0xC0000>, /* GICR */
+		      <0x0 0x52000000 0 0x2000>,  /* GICC */
+		      <0x0 0x52010000 0 0x1000>,  /* GICH */
+		      <0x0 0x52020000 0 0x20000>; /* GICV */
 		#interrupt-cells = <3>;
 		interrupt-controller;
 		interrupts = <GIC_PPI 9
-- 
1.7.9.5

